examples/experimental/frustum-cull/app.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable no-continue */
+
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import DeckGL, {SimpleMeshLayer} from 'deck.gl';
+import {StaticMap} from 'react-map-gl';
+import {SphereGeometry} from '@luma.gl/core';
+import {Vector3} from 'math.gl';
+
+const INITIAL_VIEW_STATE = {
+  latitude: 51.47,
+  longitude: 0.45,
+  zoom: 7,
+  bearing: 0,
+  pitch: 0
+};
+
+const STYLE = {
+  position: 'absolute',
+  right: '10px',
+  top: '10px',
+  padding: '10px',
+  background: 'white',
+  border: '1px solid black'
+};
+
+const position = [-0.4531566, 51.4709959, 100000];
+const radius = 10000;
+const testPosition = new Vector3();
+
+class Root extends Component {
+  constructor(props) {
+    super(props);
+    this.deckRef = React.createRef();
+    this.state = {
+      cullStatus: 'IN'
+    };
+
+    this._onViewStateChange = this._onViewStateChange.bind(this);
+  }
+
+  _onViewStateChange() {
+    const viewport = this.deckRef.current.viewports[0];
```

Comment:
`viewports` have not been updated at this point. You need to manually build the instance:

```js
import {WebMercatorViewport} from '@deck.gl/core';

_onViewStateChange({viewState}) {
  const viewport = new WebMercatorViewport(viewState);
}
```